samples/cpp/text2image/main.cpp

```diff
@@ -15,9 +44,10 @@ int32_t main(int32_t argc, char* argv[]) try {
     ov::Tensor image = pipe.generate(prompt,
         ov::genai::width(512),
         ov::genai::height(512),
-        ov::genai::num_inference_steps(20));
+        ov::genai::num_inference_steps(20),
+        ov::genai::num_images_per_prompt(1));
 
-    imwrite("image.bmp", image, true);
+    imwrite_output_imgs(image);
```

Comment:
Maybe we can change `imwrite` itself to the following signature:
```
imwrite("image_%d.bmp", image, true);
```
where `image_%d.bmp` is a pattern which can be passed to `sprintf` to create image file names ?

Reply:
Done